client/app/startup/TopbarApp.js

```diff
@@ -20,6 +20,6 @@ export default (props, marketplaceContext) => {
     'sign_up',
   ], { locale });
 
-  const combinedProps = Object.assign({}, props, { marketplaceContext, routes });
+  const combinedProps = Object.assign({}, props, { routes });
```

Comment:
I'm not sure but this change might break the landing page topbar dynamic rendering because it passes in the marketplaceContext too?